packages/Popover/src/Popover.js

```diff
@@ -181,8 +180,15 @@ class Popover extends React.Component {
     }
   }
 
+  componentWillReceiveProps(nextProps) {
```

Comment:
This lifecycle is deprecated. Perhaps in the future we can migrate this to `getDerivedStateFromProps`.